### PR TITLE
Add audio level handling, primary speaker, expose more state in Call

### PR DESCRIPTION
**Pull request by filbabic**

### CODE REVIEW

File: app/src/main/kotlin/io/getstream/video/android/app/ui/DeeplinkingActivity.kt [hunk 1]
```diff
@@ -84,7 +86,7 @@ class DeeplinkingActivity : AppCompatActivity() {
         VideoApp.initializeStream(
             credentialsProvider = FakeCredentialsProvider(
                 userCredentials = selectedUser,
-                apiKey = "key1"
+                apiKey = "key10"
```
Comment (kanat) [inline app/src/main/kotlin/io/getstream/video/android/app/ui/DeeplinkingActivity.kt:89]: Hey @filbabic,

Is this gonna work with local running servers, right?

File: stream-video-android/src/main/kotlin/io/getstream/video/android/webrtc/signal/RemoteSignalService.kt [hunk 2]
```diff
@@ -33,24 +33,24 @@ import stream.video.sfu.UpdateSubscriptionsResponse
...
     public override suspend fun join(@Body joinRequest: JoinRequest): JoinResponse
 
     @Headers("Content-Type: application/protobuf")
-    @POST("/rpc/twirp/stream.video.sfu.SignalServer/SetPublisher")
+    @POST("/twirp/stream.video.sfu.SignalServer/SetPublisher")
     public override suspend fun setPublisher(@Body request: SetPublisherRequest): SetPublisherResponse
 
     @Headers("Content-Type: application/protobuf")
-    @POST("/rpc/twirp/stream.video.sfu.SignalServer/UpdateSubscriptions")
+    @POST("/twirp/stream.video.sfu.SignalServer/UpdateSubscriptions")
```
Comment (kanat) [inline stream-video-android/src/main/kotlin/io/getstream/video/android/webrtc/signal/RemoteSignalService.kt]: Did they change the path? 🤔

Comment (filbabic) [reply stream-video-android/src/main/kotlin/io/getstream/video/android/webrtc/signal/RemoteSignalService.kt]: Yeah so it seems for this staging environment they don't require the /rpc stuff 🤷 We'll see how things go and what will change in the future

File: app/src/main/kotlin/io/getstream/video/android/app/ui/DeeplinkingActivity.kt (hunk 1 above)
Comment (filbabic) [reply app/src/main/kotlin/io/getstream/video/android/app/ui/DeeplinkingActivity.kt:89]: They should!

We have an app10 in hte default apps for local environments, so if things don't work we can check with BE

File: stream-video-android/src/main/kotlin/io/getstream/video/android/webrtc/signal/RemoteSignalService.kt (hunk 2 above)
Comment (filbabic) [reply stream-video-android/src/main/kotlin/io/getstream/video/android/webrtc/signal/RemoteSignalService.kt]: Removed for local yeah! Might change in the future

**Pull request merged**